velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -20,6 +20,36 @@
 #include "velox/type/Type.h"
 
 namespace facebook::velox::substrait {
+
+HiveConnectorHandler::HiveConnectorHandler(
+    const std::string& connectorId,
+    const std::string& tableName,
+    const bool filterPushDownEnabled,
+    const connector::hive::HiveColumnHandle::ColumnType& columnType)
+    : connectorId_(connectorId),
+      tableName_(tableName),
+      filterPushDownEnabled_(filterPushDownEnabled),
+      columnType_(columnType) {}
+
+std::shared_ptr<connector::ConnectorTableHandle>
+HiveConnectorHandler::createTableHandle(
+    connector::hive::SubfieldFilters filters) {
+  return std::make_shared<connector::hive::HiveTableHandle>(
+      connectorId_,
+      tableName_,
+      filterPushDownEnabled_,
+      std::move(filters),
+      nullptr);
+}
+
+std::shared_ptr<connector::ColumnHandle>
+HiveConnectorHandler::createColumnHandle(
```

Comment:
How would we handle queries that access both regular columns and partition keys?